### ISSUE TITLE
Install fails due to missing file or directory

### ISSUE THREAD

**Issue by sschelm** [symengine/SymEngine.jl#107]
While installing julia 0.6.2 on Mac OS 10.12.6, I got the following error:
```
LoadError: could not spawn setenv(
`/Users/user/.julia/v0.6/SymEngine/deps/usr/bin/conda config --add channels conda-forge --force`<...>
): no such file or directory (ENOENT)
```
There is no `bin` directory in the `SymEngine` package at the specified folder.
The Conda package is installed and works (tested with `using Conda`).
`Conda.runconda(`info`)` return for the channels:
```
channel URLs : https://conda.anaconda.org/conda-forge/osx-64
                          https://conda.anaconda.org/conda-forge/noarch
```
I therefore see no reason to add the forge channel.
It looks to me like the install is not using the actual Conda package.
`CONDARC `, `CONDA_DEFAULT_ENV ` and `CONDA_PREFIX ` all point to the `SymEngine` package and not the actual Conda package.

**Comment by isuruf** [comment]
This is probably an issue with the new conda 4.4 released a few hours ago.

**Comment by sschelm** [comment]
Is that the actual binary conda? `Conda.jl` is at 0.7.1

**Comment by isuruf** [comment]
Yes, actual conda.

Can you try the fix in https://github.com/symengine/SymEngine.jl/pull/108/files and report back?

**Comment by sschelm** [comment]
Yes, this works. Thanks.
```
INFO: Building Conda
INFO: Building SymEngine
Warning: 'conda-forge' already in 'channels' list, moving to the top
```

**Comment by isuruf** [comment]
Fixed in v0.3.0
